Codeinwp/neve
# chore: make sure all keys in values are defined for Spacing Component

### Summary
This should fix any errors with undefined object properties for the spacing control. We are now making sure that all needed keys are defined. 


### Will affect visual aspect of the product

NO

### Test instructions


- You can set controls to a bad value using the console after you load up the customizer by running the following command:  `wp.customize.control('logo_component_margin').setting.set({})`
- After that you should be able to save the customizer
- On the old version from `development` it should be throwing out errors (that it cannot read properties of undefined) but on this branch everything should still work fine, even though the spacing control will be reset to the default values.


Closes #1726.
